FilmFestivalLoader/IFFR/parse_iffr_html.py

```diff
@@ -6,15 +6,18 @@
 from copy import copy
 from enum import Enum, auto
 from typing import Dict
-from urllib.error import HTTPError
 
 from Shared.application_tools import ErrorCollector, DebugRecorder, comment, Config, Counter
 from Shared.parse_tools import FileKeeper, try_parse_festival_sites, HtmlPageParser
-from Shared.planner_interface import FilmInfo, Screening, ScreenedFilmType, ScreenedFilm, FestivalData, Film
-from Shared.web_tools import UrlFile, iri_slug_to_url, fix_json, get_encoding, UrlReader
+from Shared.planner_interface import FilmInfo, Screening, ScreenedFilmType, ScreenedFilm, FestivalData, Film, \
+    get_screen_from_parse_name
+from Shared.web_tools import UrlFile, iri_slug_to_url, fix_json
 
 ALWAYS_DOWNLOAD = True
 DEBUGGING = True
```

Comment:
The team allows you to keep debuging on until the festival programma is published and code is final.